fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/common/StandingInstructionsHelper.java

```diff
@@ -108,20 +108,20 @@ public Integer createStandingInstruction(final String clientId, final String fro
     public HashMap getStandingInstructionById(final String standingInstructionId) {
 
         LOG.info("----------------------------- RETRIEVING STANDING INSTRUCTION BY ID---------------------------");
-        final String GET_STANDING_INSTRUCTION_BY_ID_URL = STANDING_INSTRUCTIONS_URL + "/" + standingInstructionId + "?"
+        final String getStandingInstructionsByIDurl = STANDING_INSTRUCTIONS_URL + "/" + standingInstructionId + "?"
                 + Utils.TENANT_IDENTIFIER;
```

Comment:
Would be good to be consistent in capitalising URL - so this would be getStandingInstructionsByIdURL

Reply:
updated.